packages/scxa-cell-type-wheel/cypress/component/CellTypeWheel.cy.js

```diff
@@ -89,4 +90,26 @@ describe(`<CellTypeWheel>`, () => {
 
     cy.get(`@onCellTypeWheelClick`).should(`not.have.been.called`)
   })
+
+  it(`should show data labels properly while zooming in from an inner cell`, () => {
```

Comment:
I think this describes the test more accurately: “should show data labels properly after clicking on a species/organism part and zooming in”.